modules/ding_debt/plugins/content_types/debts/debts.inc

```diff
@@ -260,6 +258,12 @@ function ding_debt_debts_form_submit_pay_all($form, &$form_state) {
   $amount = 0;
   $debts = $form_state['values']['debt_data'];
   foreach ($debts as $debt_id => $debt_data) {
+    // Remove payments that are not payable, do to having a gateway payment
```

Comment:
I don't understand this should it be "due to having a gateway payment"?

Reply:
Updated comment.